src/resource/event/event.controller.ts

```diff
@@ -128,13 +128,13 @@ export class EventController {
     @Delete(':id')
     @DeleteEventApi()
     async deleteEvent(@Param('id') eventId: number): Promise<void> {
-        await this.eventService.deleteById(eventId);
+        await this.eventService.delete(eventId);
     }
 
     @HttpCode(HttpStatus.NO_CONTENT)
     @Delete('exceptions/:id')
     @DeleteExceptionApi()
     async deleteException(@Param('id') exceptionId: number): Promise<void> {
-        await this.eventService.removeExceptionById(exceptionId);
+        await this.eventService.deleteException(exceptionId);
```

Comment:
deleteException in this case is clear, because we know that we want to delete one so it is obvious that we need for example id